src/accelerate/utils/imports.py

```diff
@@ -38,12 +38,7 @@
 try:
     import torch_xla.core.xla_model as xm  # noqa: F401
 
-    try:
-        # Will raise a RuntimeError if no XLA configuration is found
-        _ = xm.xla_device()
-        _tpu_available = True
-    except RuntimeError:
-        _tpu_available = False
```

Comment:
Mmm, removing this will break the other places we use `is_tpu_available`

Reply:
The added reason was specifically inside `AcceleratorState`, at the proposed location. But having it as an argument instead works as well. Will refactor